src/main/scala/firrtl/transforms/ConstantPropagation.scala

```diff
@@ -676,8 +698,9 @@ class ConstantPropagation extends Transform with DependencyAPIMigration with Res
       // Actually transform some statements
       stmtx match {
         // Propagate connections to references
-        case Connect(info, lhs, rref @ WRef(rname, _, NodeKind, _)) if !dontTouches.contains(rname) =>
-          Connect(info, lhs, nodeMap(rname))
+        case Connect(info0, lhs, rref @ WRef(rname, _, NodeKind, _)) if !dontTouches.contains(rname) =>
+          val (info1, value) = unwrap(nodeMap(rname))
+          Connect(orElse(info0, info1), lhs, value)
```

Comment:
This is the opposite because I forgot to update it, nice catch. I should also have a comment pointing to that function since these kind of should be in sync

Reply:
Honestly, this code looks like it should be unreachable (the node reference should be propagated during the `constPropExpression` above), but I ran scoverage and it does seem to be hit somehow. I might mark this as a TODO check if still necessary.